docs/GENERATORS.md

```diff
@@ -87,6 +87,8 @@ Allow additional methods to be added to the test suite.
 
 There may exist cases where a suite of unit tests will need to reuse the same logic in each of the tests. Rather than duplicating code, this method allows you to provide helper methods for the tests.
 
+An example of this is the [Tournament](https://github.com/exercism/csharp/blob/master/generators/Exercises/Tournament.cs)
```

Comment:
Good addition. Two small points.

1. Could you link to the line where the method is overridden? The link is: https://github.com/exercism/csharp/blob/master/generators/Exercises/Tournament.cs#L45
1. Could you add " generator." to the end of the line, so that is reads "An example of this is the Tournament generator."?